src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -45,13 +46,22 @@
  *     filter on contamination fractions. Alternatively, provide a numerical fraction to filter with --contamination_fraction_to_filter.
  * </p>
  *
- * <h3>Example</h3>
+ * <h3>Input</h3>
+ * <p>
+ * VCF of unfiltered Mutect2 SNV and indel calls.
+ * </p>
  *
+ * <h3>Output</h3>
+ * <p>
+ * A VCF of filtered SNV and indel calls.
+ * </p>
+ *
+ * <h3>Example</h3>
  * <pre>
  * gatk-launch --javaOptions "-Xmx4g" FilterMutectCalls \
- *   -V tumor_matched_m2_snvs_indels.vcf.gz \
+ *   -V unfiltered.vcf \
  *   -contaminationTable contamination.table \
- *   -O tumor_matched_m2_filtered.vcf.gz
+ *   -O filtered.vcf
```

Comment:
gatk-launch --> gatk

While at it, 

vcf --> vcf.gz

Reply:
done x 2